engine/apps/user_management/models/user.py

```diff
@@ -272,10 +279,6 @@ def clear_phone_numbers(self) -> None:
         self._verified_phone_number = None
         self.save(update_fields=["unverified_phone_number", "_verified_phone_number"])
 
-    # TODO: move to telegram app
-    def is_telegram_connected(self):
-        return hasattr(self, "telegram_connection")
-
```

Comment:
`mypy` [caught a bug](https://github.com/grafana/oncall/actions/runs/7700348181/job/20983996764?pr=3778#step:4:535) on CI:
```
apps/api/views/user.py:603: error: Function "is_telegram_connected" could always be true in boolean context  [truthy-function]
Found 1 error in 1 file (checked 670 source files)
```

[Here](https://github.com/grafana/oncall/blob/dev/engine/apps/api/views/user.py#L603) we're doing:
```python3
if not user.is_telegram_connected:
            return Response(status=status.HTTP_400_BAD_REQUEST)
```
in its current state, `User.is_telegram_connected` is a method, not a property so `not user.is_telegram_connected` is not evaluating as we likely originally intended...

There's only one reference to this method (now a property), so this should be fine refactoring as such.